lib/Settings/SettingsSection.php

```diff
@@ -12,7 +12,7 @@
 class SettingsSection implements IIconSection {
 	public function __construct(
 		private IURLGenerator $urlGenerator,
-		private IL10N $l
+		private IL10N $l,
```

Comment:
Unnecessary change, you're not even changing anything else in this file

Reply:
I guess this is the change made from the `composer cs:fix`, shall we just leave it?